tests/integration/test_distributed_backward_compatability/test.py

```diff
@@ -7,11 +7,10 @@
 node_old = cluster.add_instance(
     "node1",
     main_configs=["configs/remote_servers.xml"],
-    image="yandex/clickhouse-server",
-    tag="20.8.11.17",
```

Comment:
But then this is not a backward compatibility check, maybe it worth to add 3-d node, with analyzer separately?

Reply:
It's still the backward compatibility check. Actually, this test can be removed completely, because `20.8` is too old and this test checks the change from `20.9`.

3-d node doesn't make sense, because the analyzer is "not backward compatible". We don't support clusters with different analyzers.